genie-web/src/main/java/com/netflix/genie/web/jpa/utils/package-info.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+/**
+ * Utilities for working with JPA or the Genie databases in general.
+ *
+ * @author tgianos
+ * @since 4.0.0
+ */
+package com.netflix.genie.web.jpa.utils;
```

Comment:
How about the usual annotations we have in other modules?

Reply:
Forgot it. Added. Thanks.